### DISCUSSION TITLE
ویرایش جداول پرباد

### DISCUSSION THREAD

**Discussion by mojtabanabavi** [Sina-Soltani/Parbad#199] [Q&A]
سلام
من برای تراکنش ها جداولی دارم. میخوام به جای اون ها از جداول پرباد استفاده کنم. حتی امکان افزودن یک رابطه از جدول پرداخت ها به جدول کاربران کارم رو راه میندازه
هدف اصلی من اینه که بتونم متوجه بشم هر تراکنشی که تو جداول پرباد ذخیره میشه متعلق به کدوم کاربره
اگر امکان انجام این کار به نحوی وجود داره لطفا منو راهنمایی کنید

**Comment by Sina-Soltani** [comment]
درود

ویژگی ویرایش جداول پَرباد، جز مواردی هست که در لیست توسعه هست و باید توسعه داده بشه
بنابراین چون فعلا موجود نیست یکی از کارهای زیر رو میتونید انجام بدید

روش اول: از جداول پَرباد اصلا استفاده نکنید و ذخیره و بازیابی رو خودتون انجام بدید
[Custom Storage](https://github.com/Sina-Soltani/Parbad/wiki/Configuration#using-a-custom-storage)

روش دوم: اگر هنوز قصد دارید از مخزن انتیتی فریم ورک پَرباد استفاده کنید، میتونید اینترفیس
IStorageManager
رو به هر کجا که خواستید تزریق کنید و ازش استفاده کنید تا بتونید رکورد های مربوط به پرداخت و تراکنش های مربوط به یک شماره پیگیری خاص رو بگیرید

چون شماره پیگیری چیزی هست که شما در تمام مراحل دارید

**Reply by mojtabanabavi** [reply]
ممنون
